source/refunding_payments/index.html.md.erb

```diff
@@ -62,7 +67,10 @@ If you needed to know the details of the partial refunds (for example, whether t
 
 ## Specifying the expected refund available
 
-When you submit a refund request for a payment via the API, you can optionally specify a ``refund_amount_available`` value in the body of the request.
+When you submit a refund request for a payment via the API, you can optionally
+specify a ``refund_amount_available`` value in the body of the request. This
+value [includes any corporate card 
+surcharges](/optional_features/corporate_card_surcharges/#corporate-card-surcharges). 
```

Comment:
It's a bit tricky without rewording this whole section (which I will do eventually), because it breaks the flow of the other paragraphs a little. I wonder if it might be best to put the note about adding corporate surcharges to the bottom of the "Specifying the expected..." section. 

Just to be 100% clear: we are talking about the `"refund_amount_available"` value in the context of including corporate card surcharges [in something]?